components/brave_federated/data_stores/data_store.cc

```diff
@@ -49,14 +49,14 @@ void BindCovariateToStatement(
     const brave_federated::mojom::CovariateInfo& covariate,
     int training_instance_id,
     base::Time created_at,
-    sql::Statement* stmt) {
-  DCHECK(stmt);
-
-  stmt->BindInt(0, training_instance_id);
-  stmt->BindInt(1, static_cast<int>(covariate.type));
-  stmt->BindInt(2, static_cast<int>(covariate.data_type));
-  stmt->BindString(3, covariate.value);
-  stmt->BindDouble(4, created_at.ToDoubleT());
+    sql::Statement* statement) {
+  CHECK(statement);
+
+  statement->BindInt(0, training_instance_id);
+  statement->BindInt(1, static_cast<int>(covariate.type));
+  statement->BindInt(2, static_cast<int>(covariate.data_type));
+  statement->BindString(3, covariate.value);
+  statement->BindDouble(4, created_at.ToDoubleT());
```

Comment:
serialized timestamps should use `.ToDeltaSinceWindowsEpoch().InMicroseconds()` and `base::Time::FromDeltaSinceWindowsEpoch(...)`